Aptoide/aptoide-client-v8
# [feature/ASV-1775] AppView Dark Theme

**What does this PR do?**

   This adds support in the AppView for dark theme.

**Database changed?**

   No

**How should this be manually tested?**

  Check both the dark and default (white) theme in App View. It should look similar to the Zeplin specification. Note that the card colors are fixed in another ticket and not in this one. I also didn't change the dialog colors.

**What are the relevant tickets?**

  [ASV-1775](https://aptoide.atlassian.net/browse/ASV-1775)


**Code Review Checklist**
